kythe/proto/storage.proto

```diff
@@ -22,6 +22,7 @@ option go_package = "storage_go_proto";
 option java_package = "com.google.devtools.kythe.proto";
 
 // VName is a proto representation of a vector name.
+// See also https://kythe.io/docs/kythe-storage.html#_a_id_termvname_a_vector_name_strong_vname_strong
```

Comment:
This seems to have upset a linter: "Error  (S&RX) clang-format" but the messages doesn't say what's wrong.
Is it a too-long line? (How can I run the linter stand-alone to check?)

Reply:
`arc lint` will run the linter provided you have Arcanist installed. It looks like you just have to run `clang-format` on the proto file.